README.md

```diff
@@ -97,6 +97,25 @@ Run the following command to setup the database once Ghostfolio is running:
 docker-compose -f docker/docker-compose.yml exec ghostfolio yarn database:setup
 ```
 
+#### For self-hosted environment (e.g Unraid/portainer)
+
+1)  Run the ghostfolio container
+2)  Open the container cli
+3)  Enter the following command ONLY the first time to create the database and relative tables.
+
+```bash
+yarn database:migrate
```

Comment:
Shouldn't it be `yarn database:setup` the first time. What do you think @LeonStoldt?

Reply:
yarn database:setup caused problems for some people as mentioned in [Issue 924](https://github.com/ghostfolio/ghostfolio/issues/924). That is why he wrote database:migrate (I think). If this issue is fixed and there are no problems with it, it could / should be the setup command). I didn't try it myself yet with setup.